src/mapml.js

```diff
@@ -3191,48 +3515,51 @@ M.MapMLStaticTileLayer = L.GridLayer.extend({
     return tileElem;
   },
 
-  //----------------------------------
-  //  minX,minY
-  // 
-  //
-  //
-  //
-  //                      maxX,maxY
-  //----------------------------------
-  //between those is the bounds of the layer
-  //gets the bounds of each zoomlevel in terms of meters
-  _getLayerBounds: function(tileGroups, resolutions){
+  _getLayerBounds: function(tileGroups, projection){
     let layerBounds = {};
     for(let tile in tileGroups){
-      let sCoords = tile.split(":"), pixelCoords = {}, zoomConstant = resolutions[sCoords[2]];
-      pixelCoords.x = +sCoords[0] * 256;
-      pixelCoords.y = +sCoords[1] * 256;
+      let sCoords = tile.split(":"), pixelCoords = {};
+      pixelCoords.x = +sCoords[0] * TILE_SIZE;
+      pixelCoords.y = +sCoords[1] * TILE_SIZE;
       pixelCoords.z = +sCoords[2]; //+String same as parseInt(String)
       if(sCoords[2] in layerBounds){
-        layerBounds[sCoords[2]].extend(L.point(pixelCoords.x * zoomConstant,pixelCoords.y * zoomConstant));
-        layerBounds[sCoords[2]].extend(L.point(((pixelCoords.x+256) * zoomConstant),((pixelCoords.y+256) * zoomConstant)));
+
+        layerBounds[sCoords[2]].extend(L.point(pixelCoords.x ,pixelCoords.y ));
+        layerBounds[sCoords[2]].extend(L.point(((pixelCoords.x+TILE_SIZE) ),((pixelCoords.y+TILE_SIZE) )));
       } else{
-        layerBounds[sCoords[2]] = L.bounds(L.point(pixelCoords.x * zoomConstant,pixelCoords.y * zoomConstant),L.point(((pixelCoords.x+256) * zoomConstant),((pixelCoords.y+256) * zoomConstant)));
+        layerBounds[sCoords[2]] = L.bounds(
+                                    L.point(pixelCoords.x ,pixelCoords.y ),
+                                    L.point(((pixelCoords.x+TILE_SIZE) ),((pixelCoords.y+TILE_SIZE) )));
       }
     }
+    // TODO: optimize by removing 2nd loop, add util function to convert point in pixels to point in pcrs, use that instead then this loop
```

Comment:
Is this something better recorded as an issue?

Reply:
Maybe, wdyt? It's referring to a for loop that can be removed by adding more code, I wasn't sure if the small optimization was worth the additional code and I'm still unsure, if you have any suggestions let me know.